ui-admin/src/api/api.tsx

```diff
@@ -487,6 +487,15 @@ export default {
     return await this.processJsonResponse(response)
   },
 
+  async deleteStudy(portalShortcode: string, studyShortcode: string): Promise<Response> {
+    const url = `${API_ROOT}/portals/v1/${portalShortcode}/studies/${studyShortcode}`
```

Comment:
we've got a helper function for some of this url
```suggestion
    const url = `${basePortalUrl(portalShortcode)}/studies/${studyShortcode}`
```